module/Application/view/error/404.phtml

```diff
@@ -1,113 +1,105 @@
 <h1>A 404 error occurred</h1>
 <h2><?= $this->message ?></h2>
 
-<?php if (isset($this->reason) && $this->reason): ?>
-
-<?php
-$reasonMessage= '';
-switch ($this->reason) {
-    case 'error-controller-cannot-dispatch':
-        $reasonMessage = 'The requested controller was unable to dispatch the request.';
-        break;
-    case 'error-controller-not-found':
-        $reasonMessage = 'The requested controller could not be mapped to an existing controller class.';
-        break;
-    case 'error-controller-invalid':
-        $reasonMessage = 'The requested controller was not dispatchable.';
-        break;
-    case 'error-router-no-match':
-        $reasonMessage = 'The requested URL could not be matched by routing.';
-        break;
-    default:
-        $reasonMessage = 'We cannot determine at this time why a 404 was generated.';
-        break;
-}
-?>
-
-<p><?= $reasonMessage ?></p>
-
+<?php if (!empty($this->reason)): ?>
```

Comment:
... for consistencies with ...? I can't see in any other views loops or conditional statements (where php is mixed with html). And it was like that before `if...endif`.
I've ran PHPCS with PSR-2 standards on views, and if I add spaces after parenthesis (before `:`) everything looks fine. I'm fine with change everything to `{` but then we have decide that is new standard from now and maybe add some PSR-2 checks/auto fixes also on views?

BTW. Is zend using PSR-2 at all? I'm not sure, because of (for example): https://github.com/zendframework/zend-mvc/blob/master/.php_cs#L13